examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -97,7 +97,11 @@
     "from dask_saturn import SaturnCluster\n",
```

Comment:
<p>I don't think this is correct. It doesn't look like we use numbers like this on <code>main</code>  <a href="https://github.com/saturncloud/examples/tree/main/examples/examples-gpu/nyc-taxi-snowflake" target="_blank">https://github.com/saturncloud/examples/tree/main/examples/examples-gpu/nyc-taxi-snowflake</a>, or in this PR. Can you revert this?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>I think I just copied that over from the CPU notebooks. I'll remove them, its probably best not to include hard runtime numbers because maybe it'll be slightly different for different runs</p>

Reply:
wait I'm not talking about "runtime numbers". I mean using `./01-rf-scikit.ipynb`, when the notebook is called `rf-scikit.ipynb`

Reply:
oh oh! yeah thats a mistake